src/content/troubleshooting/_index.en.md

```diff
@@ -0,0 +1,140 @@
++++
+title = "Troubleshooting Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 20
```

Comment:
This weight will be correct with https://github.com/submariner-io/submariner-website/pull/134 - should rebase on top of that (#134 is ready too merge anyay).

Reply:
Done. Check the order. I'm going with weight = order*5